make-release.sh

```diff
@@ -395,13 +404,17 @@ bumpVersion() {
 }
 
 updateImageTagsInCheServer() {
-    cd che-server
-    git checkout ${BRANCH}
-    cd .ci
-    ./set_tag_version_images.sh ${CHE_VERSION}
-    cd ..
-    git commit -asm "chore: Set ${CHE_VERSION} release image tags"
-    git push origin ${BRANCH}
+    pushd che-server >/dev/null
+        git checkout ${BRANCH}
+        plugin_version="latest"
+        sed_in_place -r -e "s#che.factory.default_editor=eclipse/che-theia/.*#che.factory.default_editor=eclipse/che-theia/$plugin_version#g" assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties
```

Comment:
this seems wrong since theia is no longer the default editor.

Reply:
I'd leave this as is, is so we could merge the fix for release, and then change it with the rest in a separate issue/PR (probably even this one https://github.com/eclipse-che/che-server/pull/393/files)